tensorflow_addons/image/median_filter_2D.py

```diff
@@ -0,0 +1,86 @@
+import tensorflow as tf
+from tensorflow.python.ops.image_ops_impl import _Assert3DImage
+
+
+
+def median_filter_2D(input, filter_shape=(3, 3)):
+    """This method performs Median Filtering on image.Filter shape can be user given.
+       This method takes both kind of images where pixel values lie between 0 to 255 and where it lies between 0.0 and 1.0
+       Args:
+           input: A 3D `Tensor` of type `float32` or 'int32' or 'float64' or 'int64 and of shape`[rows, columns, channels]`
+
+           filter_shape: Optional Argument. A tuple of 2 integers .(R,C).R is the first value is the number of rows in the
+           filter and C is the second value in the filter is the number of columns in the filter. This creates a filter of
+           shape (R,C) or RxC filter. Default value = (3,3)
+
+        Returns:
+            A 3D median filtered image tensor of shape [rows,columns,channels] and type 'int32'. Pixel value of returned
+            tensor ranges between 0 to 255
+    """
+
+    if not isinstance(filter_shape, tuple):
+        raise TypeError('Filter shape must be a tuple')
+    if len(filter_shape) != 2:
+        raise ValueError('Filter shape must be a tuple of 2 integers .Got %s values in tuple' % len(filter_shape))
+    filter_shapex = filter_shape[0]
+    filter_shapey = filter_shape[1]
+    if isinstance(filter_shapex, int) and isinstance(filter_shapey,
+            int):
+        pass
+    else:
+        raise TypeError('Size of the filter must be Integers')
```

Comment:
Cleaner to write this block as `if not isinstance()` -> raise

Reply:
fixed